yolapy/resources/site.py

```diff
@@ -0,0 +1,94 @@
+class SiteResourceMixin(object):
+
+    """Methods for managing Site resources."""
+
+    def list_sites(self, **options):
+        """Return paginated list of sites.
+
+        `yola.list_sites()`
+
+        Example response:
+
+        ```
+        {
+            'count': 999,
+            'previous': None,
+            'next': 'https://wl.qa.yola.net/pr/sites/?page=2',
+            'results': [{...}, {...}, ...]
+        }
+        ```
+
+        You can pass `page_size` and `page` as keyword arguments:
+
+        `yola.list_sites(page_size=50, page=2)`
+
+        You can also pass filters and odering options as keyword arguments.
+        See <https://wl.qa.yola.net/sites/> for available options.
+        """
+        return self.get(self._site_path(), params=options).json()
+
+    def get_site(self, site_id):
+        """Return details for a particular site.
+
+        ```
+        site = yola.get_site('site_id')
+        site['name'] # => 'My Site'
+        ```
+        """
+        return self.get(self._site_path(site_id)).json()
+
+    def disable_site(self, site_id):
+        """Disable a site.
+
+        `yola.disable_site('site_id')`
+        """
+        self.post(self._site_path(site_id, 'disable'))
+
+    def enable_site(self, site_id):
+        """Enable a site.
+
+        `yola.enable_site('site_id')`
+        """
+        self.post(self._site_path(site_id, 'enable'))
+
+    def change_site_owner(self, site_id, new_user_id):
+        """Change site owner.
+
+        `yola.change_site_owner('site_id', 'new_user_id')`
+        """
+        data = {'id': new_user_id}
+        self.post(self._site_path(site_id, 'change_owner'), data=data)
+
+    def change_partner_domain(self, site_id, new_domain):
+        """Change site's partner domain.
```

Comment:
hrm, "partner domain" probably shouldn't have got this far out. From the partner's point of view, this is just the site's domain.